### DISCUSSION TITLE
Found new channels on Discord related to popular collections

### DISCUSSION THREAD

**Discussion by yangwao** [kodadot/nft-gallery#1926] [General]
I've got the great idea to create channels related to popular collections or artists on our Discord so we can have more contextual discussions.

@cryptodamsky what's your opinions or perspective? I've found that for example mousedao has it for example tho :)

**Comment by damskyftw** [comment]
> I've got the great idea to create channels related to popular collections or artists on our Discord so we can have more contextual discussions.
> 
> @cryptodamsky what's your opinions or perspective? I've found that for example mousedao has it for example tho :)

I kinda like it, although i would like to establish a little more meaningfull connection with certain projects and artists :) since mousedao contacted us and i quite like the initiative from them, we could try some beta with them :)

Since we will be creating a category probably for this and also their own channel where they can connect, i'd like to have some space on the project's discord also. For example:

- Let's say we would establish this channel for mouseDAO and they can freely communicate and promote their projects and so on in this perticullar channel. That's nice.
- I'd also like to come to their discord, where we would have something like "KodaDot helpdesk" "KodaDot space" or something of that sort, so people who stumble upon KodaDot on mouseDAO, but, are not on our discord, can recieve some form of support from our community/community managers if they need help with whatever on KodaDot and so on.
- also a possibility, to gain a specific role on project's discord, to show our support and be there for some needs? :)

It would be sort of a connected friendly collaboration, a win win collab.

Of course, this channel for perticullar project can exist seamlessly without our presence on project's discord, i just like to create meaningfull connections, we can definitely even create some channels for some projects/artists and only offer this as possiblity :)


wdyt mousedao and others? let us know :)
